layers/+lang/bibtex/README.org

```diff
@@ -82,3 +117,24 @@ key binding added:
 | Key binding | Description     |
 |-------------+-----------------|
 | ~SPC m i c~ | Insert citation |
+
+*EBIB*
```

Comment:
analogous to insert line / delete line, we can have
"o" and "d" for add / delete entry.

Evil collections have "d" and "D" for delete/kill entries. We can follow that.
https://github.com/emacs-evil/evil-collection/blob/6c97b0c18678fdb1d6370c30324bc8df9a8fb1e1/modes/ebib/evil-collection-ebib.el#L42-L43

Reply:
This key binding is to insert a citation into org/latex/markdown buffer. I really think `, i` makes more sense here.
Also `d` is bound to delete entry already in the ebib index buffer. But it is probably handy to provide that information here also (instead of only in the ebib documentation / major-mode-describe).